src/main/java/com/github/msemys/esjc/ssl/SslSettings.java

```diff
@@ -21,22 +23,29 @@
     public final String certificateCommonName;
 
     /**
-     * Whether or not to validate the server SSL certificate.
+     * The file containing the signing certificate (chain) to validate against.
+     */
+    public final String certificateFile;
```

Comment:
suggesting api change
```suggestion
    public final File certificateFile;
```